### PR TITLE
[POC] Add workflows for accessibility

### PR DESCRIPTION
**Pull request by csouchet**

### CODE REVIEW

**Comment by csouchet** [comment]
> .flDDCN.Link__LinkInButton-sc-1epbwti-1.jeiKGq

image


The `Demo` button failed.


image

File: .github/workflows/lighthouse.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+jobs:
+  Lighthouse-CLI:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      # Using the Lighthouse GitHub Action
+      # @see https://github.com/marketplace/actions/lighthouse-ci-action
+      - name: Run Lighthouse GitHub Action
+        uses: treosh/lighthouse-ci-action@v9
```
Comment (tbouffard) [inline .github/workflows/lighthouse.yml:27]: ❓ could we also check https://github.com/foo-software/lighthouse-check-action which seems to offer more options? in another workflow for example.

File: src/components/seo.tsx [hunk 2]
```diff
@@ -58,35 +58,37 @@ export const SEO = ({
...
+        <title>{seo.title}</title>
+        <meta charSet="utf-8" />
+        <meta name="description" content={seo.description} />
+        <meta name="image" content={seo.image} />
+        <meta name="keywords" content={seo.metaKeywords} />
 
-      <meta name="theme-color" content={theme.colors.background} />
-      <meta itemProp="name" content={seo.title} />
-      <meta itemProp="description" content={seo.description} />
+        <meta name="theme-color" content={theme.colors.background} />
```
Comment (tbouffard) [inline src/components/seo.tsx:69]: ❓ Is this change related to the POC? (just for my personal knowledge)

Comment (csouchet) [reply src/components/seo.tsx:69]: It's a fix due to the accessibility report.
If it'll work, I'll report it in another PR 😉

**Comment by csouchet** [comment]
### Result

Never returns any errors :
- ♿️ A11Y CI with a11ytester-action / a11ytester-action-CLI 
- ♿️ A11Y CI with a11ywatch / a11ywatch-CLI (pull_request) 

For the others, they are complementary. Sometimes, there are the same errors. Sometimes, there are different errors.

**Comment by csouchet** [comment]
replaced by #1161, #1162, #1163, #1164, #1165, #1166

**Pull request closed without merging**